### ISSUE TITLE
Review annotations to GO:0036473 cell death in response to oxidative stress and children (non programmed)

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#4417]
Dear all, 

The proposal has been made to obsolete  the following terms:

NO ANNOTATIONS GO:1903201    regulation of oxidative stress-induced cell death

7 GO:1903209    positive regulation of oxidative stress-induced cell death
(3)	ARUK-UCL
~(3)	RGD~
~(1)	UniProt~

7 GO:1903202    negative regulation of oxidative stress-induced cell death
(2)	BHF-UCL
(2)	ParkinsonsUK-UCL
(1)	ARUK-UCL
~(1)	RGD~
~(1)	UniProt~

10 GO:1905206    positive regulation of hydrogen peroxide-induced cell death
~(5)	RGD~
(4)	BHF-UCL
~(1)	SGD~

GO:1903205    regulation of hydrogen peroxide-induced cell death
~(2)	UniProt~
~(1)	SGD~

18 GO:1903206    negative regulation of hydrogen peroxide-induced cell death
(9)	ParkinsonsUK-UCL
~(4)	RGD~
~(4)	UniProt~
(1)	BHF-UCL

26 GO:1903204    negative regulation of oxidative stress-induced neuron death
(25)	ParkinsonsUK-UCL
~(17)	UniProt~
~(13)	RGD~
(8)	MGI
(6)	ARUK-UCL
(5)	BHF-UCL
~(1)	FlyBase~
~(1)	SGD~

8 GO:1903208    negative regulation of hydrogen peroxide-induced neuron death
(5)	ParkinsonsUK-UCL
(1)	ARUK-UCL
~(1)	RGD~
~(1)	UniProt~

3 GO:1903205    regulation of hydrogen peroxide-induced cell death
~(2)	UniProt~
~(1)	SGD~

2 GO:1903203    regulation of oxidative stress-induced neuron death
~(2)	UniProt~

1 GO:1903207    regulation of hydrogen peroxide-induced neuron death
(1)	Complex Portal

7 GO:0036475 neuron death in response to oxidative stress  
(5)	MGI
(1)	ARUK-UCL
~(1)	RGD~

3 GO:0036474 cell death in response to hydrogen peroxide
~(2)	UniProt~
~(1)	RGD~
 
2 GO:0036473 cell death in response to oxidative stress 
(1)	ParkinsonsUK-UCL
~(1)	UniProt~

7 GO:1903223 positive regulation of oxidative stress-induced neuron death
(3)	ARUK-UCL
~(3)	RGD~
~(1)	UniProt~

3 GO:1905206 positive regulation of hydrogen peroxide-induced cell death
(2)	BHF-UCL
~(1)	RGD~

1 GO:1903209	positive regulation of oxidative stress-induced cell death
(1)	ParkinsonsUK-UCL

1 GO:1903207 regulation of hydrogen peroxide-induced neuron death
(1)	Complex Portal	

see https://github.com/geneontology/go-ontology/issues/24652

Experimental annotations that need to be reviewed are here: 
https://docs.google.com/spreadsheets/d/1IW38iS7D2am17r5SHmyT7ti0hq1ipTAlYTJdGb95ye4/edit#gid=0

Impacted groups: 
See above

Note that most look as though they are programmed cell death, and will move to an appropriate term in the programmed cell death branch. However some are antioxidants are required for responses to oxidative stress and their absence results in cell death when exposed to toxic levels of oxidative stressors.

Thanks.

**Comment by hdrabkin** [comment]
@ValWood  if you don't assign someone in MGI then this would have slipped through our fingers.  I just assigned me for temp now before I pass it to someone else.

**Comment by srengel** [comment]
SGD done.

**Comment by gthayman** [comment]
RGD done.

**Comment by hattrill** [comment]
FB done

**Comment by Antonialock** [comment]
uniprot done

**Comment by ValWood** [comment]
@LiNiMGI  there might be some MGI remaining here

**Comment by LiNiMGI** [comment]
@ValWood Those all have been done through ticket 4426. Thanks!

**Comment by ValWood** [comment]
Remaining
12 ARUK-UCL
9 BHF-UCL
27 Parkinsons-UCL

**Comment by RLovering** [comment]
Hi 
I have checked all of the UCL annotations to the terms listed above. I downloaded the annotations from QuickGO as sometimes some of the annotations are missing from your spreadsheet and it saves me adding them. There is some overlap between the two lists, but some of the annotations on your sheet had already changed in QuickGO so they are not on mine.
I also noticed that you probably want to change all the neuron death annotations? UCL has 64 of these so I will do these another day. Did you have a ticket for these?

Best

Ruth
